CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-api-python3#101](https://github.com/cyberark/conjur-api-python3/issues/101)
 - The `user` methods rotate-api-key and change-password are now available in CLI and API to manage users
   [cyberark/conjur-api-python3#101](https://github.com/cyberark/conjur-api-python3/issues/101)
-- Add ability to get previous variable/secret versions. This is available in both CLI and API.
+- Add ability to get previous variable/secret versions. This is available in both CLI and API
```

Comment:
please add a link to an issue.

also - what does "Available in the API" mean? The Conjur REST API?

Reply:
the SDK. will update my entries -> SDK